bin/check_go_version

```diff
@@ -2,7 +2,7 @@
 
 set -eu -o pipefail
 
-VERSION="go1.11"
+VERSION="go1.12.2"
```

Comment:
How about just `1.12`? and leave the patches alone.

Reply:
Fixed! But it looks like CircleCI is unhappy. :(